rnns/fastrnns/custom_lstms.py

```diff
@@ -246,12 +252,20 @@ def init_stacked_lstm(num_layers, layer, first_layer_args, other_layer_args):
 
 
 class StackedLSTM(jit.ScriptModule):
-    __constants__ = ['layers']  # Necessary for iterating through self.layers
+    # Necessary for iterating through self.layers and dropout support
+    __constants__ = ['layers', 'num_layers', 'dropout']
```

Comment:
I know this is bad from a software engineering perspective, but could you create a separate `StackedLSTMWithDropout?` I'd like to keep a vanilla "StackedLSTM" around (so that it does not get too complicated)